WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1670,3 +1670,32 @@ yellowgreen =           #9acd32
 VeilOverlayC1       =   @AltC@
 VeilOverlayC2       =   @darken(AltC,20)@
 VeilOverlayGrad     =   @grad(VeilOverlayC1,VeilOverlayC2,linear-vertical)@
+
+######################################################################
+# SEARCH TOOLBAR 
+######################################################################
+
+SearchResultToolbar             = background-color:@AppC@; @BottomSeparator@; padding: 0 @SkinWrapperPadding@; box-sizing: border-box; 
+SearchResultInputField          = padding: 0 @SkinWrapperPadding@; width:100%; position: relative;
+SearchResultInputFieldIcon      = position: absolute; top: 50%; margin-top: -10px; right: 24px;
+SearchResultText                = @Text-light@ @FontSize-slightly-bigger@ white-space:nowrap; @AlignRight@ vertical-align:middle; width:15%; min-width:200px;
+SearchResultButton              = display: table-cell;
+SearchFilterContainer           = background:@white@; @roundCorners(2px)@; border-color: @PanelBorderColor@;
+SearchFilter                    = height: 3rem; line-height: 2.8rem; padding: 4px 4px 0;
+SearchFilterPanel               = @OverviewTreeContainer@; border:0; width:100%; height:100%;
+SearchToolbarHeader             = @TreeHeaderContainer@; @TreeHeaderText@ @TopSeparator@; height: auto; line-height: 3rem; padding:0 @SkinWrapperPadding@;
+SearchToolbarFilter             = padding:0 @SkinWrapperPadding@; height: 3rem;
+SearchToolbarFilterText         = @FontSize-slightly-big@; fill: @PrimaryBrandingColor@
```

Comment:
`fill` is for targeting SVG icon in that container, and I don't feel need to have other specific property for that. However for readability purpose I have moved `fill` property to `SearchToolbarFilter` property and will work as before.

Thanks for it.